firmware/src/network_model.c

```diff
@@ -200,10 +200,7 @@ int NetworkModel_GetListInterfaces(const RDMHeader *header,
     ptr = PushUInt32(ptr, INTERFACE_DEFINITIONS[i].id);
     ptr = PushUInt16(ptr, INTERFACE_DEFINITIONS[i].hardware_type);
   }
-  RDMResponder_BuildHeader(header, ACK, GET_COMMAND_RESPONSE,
-                           ntohs(header->param_id),
-                           (ptr - g_rdm_buffer) - sizeof(RDMHeader));
-  return RDMUtil_AppendChecksum(g_rdm_buffer);
+  return RDMResponder_AddHeaderAndChecksum(header, ACK, ptr - g_rdm_buffer);
```

Comment:
Can we have a BuildGetAck please like BuildSetAck, which calls RDMResponder_AddHeaderAndChecksum.